components/icu/src/lib.rs

```diff
@@ -399,11 +399,10 @@ pub mod properties {
     //!
     //! // An individual enumerated property value as a `UnicodeSet`
     //!
-    //! let payload =
-    //!     sets::get_for_general_category(&provider, GeneralCategory::LineSeparator)
-    //!         .expect("The data should be valid");
+    //! let payload = maps::get_general_category(&provider).expect("The data should be valid");
     //! let data_struct = payload.get();
-    //! let line_sep = &data_struct.inv_list;
+    //! let gc = &data_struct.code_point_trie;
+    //! let line_sep = gc.get_set_for_value(GeneralCategory::LineSeparator);
```

Comment:
I'm not sure if this belongs in this patch or somewhere else, but the API we expose for this should take `GeneralCategoryGroup`, not `GeneralCategory`, to make it possible for people to ask for `gc=Letter` and so on.

Reply:
This method `get_set_for_value()` is implemented on a CodePointTrie, so it exists for all enumerated properties.

The special method that exists for the `GeneralCategoryGroup` case is over in `icu_properties::sets` called `get_for_general_category_group()`.